pkg/guacrest/openapi.yaml

```diff
@@ -0,0 +1,118 @@
+#
+#  Copyright 2024 The GUAC Authors.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+openapi: "3.0.0"
+paths:
```

Comment:
QQ: What is required to generate this - is it covered by `make generate` so it will be caught by CI if it is not the latest generation? Any additional requirements to add to [dev docs](https://docs.guac.sh/dev-setup/#ensure-you-have-the-following-tools-installed-in-your-environment) and the [Pull Request template](https://github.com/guacsec/guac/blob/main/.github/PULL_REQUEST_TEMPLATE.md)

Reply:
Yup, `make generate` covers this so it should be picked up by the CI. The `go generate` commands are in https://github.com/guacsec/guac/blob/e0407c50370dd89765f0a0e4e951fcdb69597a1b/pkg/guacrest/openapi.go. 

I updated the pull request template, and I'll send a PR to update the dev docs.